pages/new_homes_page.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "./page.rb"
+
+class NewHomesPage < Page
+
+    # UI Mapping
+    FIRST_NAME           = { :name => "first_name" }
+    LAST_NAME            = { :name => "last_name"  }
+    PHONE_NUMBER         = { :name => "number" }
+    PHONE_NUMBER_TYPE    = { :name => "number_type" }
+    OWNER_ADDRESS        = { :name => "address" }
+    CALL_WAIVER          = { :css => "label.btn:nth-child(2)" }
+    COLAPSE              = { :css => "button.collapse-person:nth-child(1)" }
+    CSS_BUTTON           = { :css => "button.btn:nth-child(2)" }
+    ZIPCODE              = { :name => "zip" }
+    ADDRESS              = { :id => "searchable-address1" }
+    SUBMIT_BUTTON        = { :xpath => "//button[@class='btn btn-primary pull-right']" }
+    SECOND_HOMEOWNER_ADD = { :css => ".add-resident" }
+    # I am sin I have no idea what this is
+    UMMM_FORGOT          = { :css => "div.mb-3:nth-child(2) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(8) > div:nth-child(1) > div:nth-child(1) > select:nth-child(2)" }
+
+    def initialize url:, driver:, wait:
+        super
+        puts "navigating to new homeowner info page"
+        navigate_to @url
```

Comment:
This is where the path should be hard coded to "home_owners/new"

Reply:
I have it in corporate events page as @path = "/corporate_events/new"
and then navigate_to @url + @path

something like that